frontend/public/components/api-explorer.tsx

```diff
@@ -101,7 +101,7 @@ const APIResourceLink_: React.FC<APIResourceLinkStateProps & APIResourceLinkOwnP
   const to = getAPIResourceLink(activeNamespace, model);
   return (
     <span className="co-resource-item">
-      <span className="co-resource-icon--fixed-width hidden-xs">
+      <span className="co-resource-icon--fixed-width pf-v6-u-display-none pf-v6-u-display-flex-on-sm">
```

Comment:
resource icon on API Explorer list page is still visible on smallest width
<img width="911" height="768" alt="Screenshot 2025-07-11 at 11 45 12 AM" src="https://github.com/user-attachments/assets/f579dc4c-1052-4334-878f-1cdbb9b6ca17" />